wgpu-core/src/identity.rs

```diff
@@ -47,10 +57,11 @@ pub(super) struct IdentityValues {
 }
 
 impl IdentityValues {
-    /// Allocate a fresh, never-before-seen id with the given `backend`.
+    /// Allocate a fresh, never-before-seen id.
```

Comment:
style: You've already used `ID` casing before, let's be consistent with that.


```suggestion
    /// Allocate a fresh, never-before-seen ID.
```